Lib/subprocess.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *, dwFlags=0, hStdInput=None, hStdOutput=None,
             self.hStdOutput = hStdOutput
```

Comment:
You're absolutely right. I was only thinking narrowly about a user passing in a handle list without considering the context. 

Can you update the docs for `handle_list` to note that the handles in the list have to be made inheritable via `os.set_handle_inheritable`, else `OSError` will be raised for the error code `ERROR_INVALID_PARAMETER` (87). Otherwise users may expect it to work like the `pass_fds` parameter. Including the error code in the docs may help with searching, since it's such a generic error. 

Do you think it needs a warning that concurrent calls to `CreateProcess` that inherit all handles may inadvertently inherit the handles in this list? The race condition can only be avoided with careful design, such as by always using `Popen` with a handle list and avoiding concurrent calls to `os.system` and `os.spawn*` or by using a lock to synchronize process creation.